CMakeLists.txt

```diff
@@ -7,6 +7,6 @@ project(dummy
 
 include(GNUInstallDirs)
 
-install(FILES dummy-config.cmake my-super-cmake-file.cmake
-	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
-)
+#install(FILES dummy-config.cmake my-super-cmake-file.cmake
```

Comment:
I have commented this as now there are no configuration files to install. Here you should install your binaries and headers

Reply:
In my real application, dummy-config.cmake is generated and contains some really useful CMake variables for other projects that would depend on it. I still need to install them

Reply:
I understand. However, it might be possible to add that information in the conanfile.py package_info() method providing the needed information to consumers